webapp/src/index.html

```diff
@@ -2,13 +2,28 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Webapp</title>
+  <title>Communication project</title>
   <base href="/">
-
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+  <link rel="icon" href="http://www.invivoo.com/wp-content/uploads/cropped-favicon-invivoo-32x32.png" sizes="32x32">
+<!-- DevExtreme dependencies -->
+<script type="text/javascript" src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.3.min.js"></script>
+<!-- The Knockout library -->
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/knockout/3.4.0/knockout-min.js"></script>
+<!-- A DevExtreme library -->
+<script type="text/javascript" src="https://cdn3.devexpress.com/jslib/16.1.14/js/dx.all.js"></script>
+<!--<script type="text/javascript" src="https://cdn3.devexpress.com/jslib/16.1.14/js/dx.web.js"></script>-->
```

Comment:
whyN

Reply:
I migrated bootstrap dependency to cli but devexpress dependencies it's not possible to change them